libs/types/src/locations.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+use codec::{Decode, Encode, MaxEncodedLen};
+use frame_support::RuntimeDebugNoBound;
+use scale_info::TypeInfo;
+use sp_core::{H160, H256};
+use sp_runtime::{
+	traits::{BlakeTwo256, Hash},
+	AccountId32,
+};
+use xcm::{v1::MultiLocation, VersionedMultiLocation};
+
+use crate::domain_address::DomainAddress;
+
+/// Location types for destinations that can receive restricted transfers
+#[derive(Clone, RuntimeDebugNoBound, Encode, Decode, Eq, PartialEq, MaxEncodedLen, TypeInfo)]
+pub enum Location {
+	/// Local chain account sending destination.
+	Local(AccountId32),
+	/// Test
+	TestLocal(u64),
+	/// XCM MultiLocation sending destinations.
+	/// Using hash value here as Multilocation is large -- v1 is 512 bytes, but next largest is only 40 bytes
+	/// other values aren't hashed as we have blake2 hashing on storage map keys, and we don't want the extra overhead
+	XCM(H256),
+	/// DomainAddress sending location from connectors
+	Address(DomainAddress),
+	/// Ethereum address, for cases where we would have a standalone Eth address
+	Eth(H160),
```

Comment:
I would also remove it as we won't have native H160 support. At least not that I can think of.

Reply:
👍 removed